x-pack/docs/en/rest-api/security/invalidate-tokens.asciidoc

```diff
@@ -1,31 +1,37 @@
 [role="xpack"]
 [[security-api-invalidate-token]]
-=== Delete token API
+=== Invalidate token API
 
-Invalidates a bearer token for access without requiring basic authentication.
+Invalidates an access token or a refresh token.
 
 ==== Request
 
 `DELETE /_xpack/security/oauth2/token`
 
 ==== Description
 
-The tokens returned by the <<security-api-get-token,get token API>> have a 
+The access tokens returned by the <<security-api-get-token,get token API>> have a
 finite period of time for which they are valid and after that time period, they 
 can no longer be used. That time period is defined by the 
 `xpack.security.authc.token.timeout` setting. For more information, see 
 <<token-service-settings>>.
 
-If you want to invalidate a token immediately, use this delete token API.
+The refresh tokens returned by the <<security-api-get-token,get token API>> are
+only valid for 24 hours. They can also be used exactly once.
+
+If you want to invalidate an access or refresh token immediately, use this invalidate token API.
 
 
 ==== Request Body
 
 The following parameters can be specified in the body of a DELETE request and
-pertain to deleting a token:
+pertain to invalidating a token:
+
+`token` (optional)::
+(string) An access token. This parameter cannot be used when `refresh_token` is used.
 
-`token` (required)::
-(string) An access token.
+`refresh_token` (optional)::
+(string) A refresh token. This parameter cannot be used when `token` is used.
 
```

Comment:
It's obvious to us, but sometimes things are not so obvious to others.... should we mention that one of them is required?
_Although I see, technically, if you don't specify either, then it's just a no-op action_.

Reply:
Yes, I also contemplated how this should be written and didn't find any examples of mutually excluded parameters for any of our APIs.. I'd better add a NOTE and clarify this